tests/Functional/Action/Lti/UpdateLtiOutcomeActionTest.php

```diff
@@ -28,27 +28,36 @@
 use App\Repository\InfrastructureRepository;
 use App\Security\OAuth\OAuthContext;
 use App\Security\OAuth\OAuthSigner;
-use App\Tests\Traits\DatabaseFixturesTrait;
+use App\Tests\Traits\DatabaseTestingTrait;
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
```

Comment:
Why should we have it here, only for line 40? Or am I missing smth?

Reply:
Sorry I don't get the question